ompi/mca/sharedfp/individual/sharedfp_individual.c

```diff
@@ -87,32 +84,33 @@ struct mca_sharedfp_base_module_1_0_0_t * mca_sharedfp_individual_component_file
     amode = fh->f_amode;
     if ( amode & MPI_MODE_WRONLY || amode & MPI_MODE_RDWR ) {
         wronly_flag=true;
-	if ( mca_sharedfp_individual_verbose ) {
+        if ( mca_sharedfp_individual_verbose ) {
             opal_output(ompi_sharedfp_base_framework.framework_output,
                         "mca_sharedfp_individual_component_file_query: "
                         "MPI_MODE_WRONLY[true=%d,false=%d]=%d\n",true,false,wronly_flag);
-	}
+        }
     } else {
         wronly_flag=false;
-	if ( mca_sharedfp_individual_verbose ) {
+        if ( mca_sharedfp_individual_verbose ) {
             opal_output(ompi_sharedfp_base_framework.framework_output,
-			"mca_sharedfp_individual_component_file_query: Can not run!, "
-			"MPI_MODE_WRONLY[true=%d,false=%d]=%d\n",true,false,wronly_flag);
-	}
+                        "mca_sharedfp_individual_component_file_query: Can not run!, "
+                        "MPI_MODE_WRONLY[true=%d,false=%d]=%d\n",true,false,wronly_flag);
+        }
     }
 
     /*---------------------------------------------------------*/
     /* 2. Did the user specify MPI_INFO relaxed ordering flag? */
     info = fh->f_info;
     if ( info != &(MPI_INFO_NULL->super) ){
-        valuelen = MPI_MAX_INFO_VAL;
-        opal_info_get ( info,"OMPIO_SHAREDFP_RELAXED_ORDERING", valuelen, value, &flag);
+        int flag;
+        char value[MPI_MAX_INFO_VAL+1];
```

Comment:
at the other ompio components you changed the valuelen from MPI_MAX_INFO_VAL to 128, shouldn't we do the same here?

Reply:
That case seemed special to me: the value is not actually parsed but the mere existance of the key seems to be sufficient. The value is printed using `opal_output` so I didn't want to touch that.